lib/relooper/src/graph/caterpillar.rs

```diff
@@ -37,23 +52,23 @@ pub fn unfold_dyn_edges<T: CfgLabel>(cfg: Cfg<EvmLabel<T>>) -> Cfg<CaterpillarLa
         [single] => {
```

Comment:
Maybe it will be better to check size of ``jumpdests`` before ``dyn_nodes`` initialization and save some CPU cycles? But overall it's not important, I think it works

Reply:
I even think readability here is more important then performance